### PR TITLE
201 ux forms

### PR DESCRIPTION
**Pull request by LaksR**
Work is still in progress. Please comment if the approach is right. Thanks.

### CODE REVIEW

File: src/components/ux-radio/ux-radio.hbs [hunk 1]
```diff
@@ -0,0 +1,12 @@
+
+    <label>{{.label}}</label>
+    {{#each items}}
+        <input {{#if .type}}type="{{.type}}"{{/if}}
+            {{#if .name}}name="{{.name}}"{{/if}}
+            {{#if .id}}id="{{.id}}"{{/if}}
+            {{#if .value}}value="{{.value}}"{{/if}}
+               on-blur="handleBlur"
+               on-tap="handleTap">
```
Comment (nathf) [inline src/components/ux-radio/ux-radio.hbs:9]: If these are validation purposes you can merge them like `on-blur-tap="validate"` or something similar.

Comment (ScottMaclure) [inline src/components/ux-radio/ux-radio.hbs:9]: Also use semantic names for events. Name an event by its "intent" rather
than concrete action.

See ractive docs for details on event naming.

**Comment by eneo5541** [comment]
Is it necessary to include the ux-form-label components? They don't seem to be used anywhere.

File: src/components/ux-forms/manifest.json [hunk 2]
```diff
@@ -0,0 +1,10 @@
+{
```
Comment (eneo5541) [inline src/components/ux-forms/manifest.json:1]: Manifest should be updated from boilerplate to reflect each data property used by the component

File: src/components/ux-forms/ux-forms.feature [hunk 3]
```diff
@@ -0,0 +1,5 @@
+Feature: ux-forms test suite
```
Comment (eneo5541) [inline src/components/ux-forms/ux-forms.feature:1]: There should be some tests written for this component.

File: src/components/ux-radio/ux-radio.feature [hunk 4]
```diff
@@ -0,0 +1,5 @@
+Feature: ux-radio test suite
```
Comment (eneo5541) [inline src/components/ux-radio/ux-radio.feature:1]: Again, should have tests for this component

File: src/components/ux-select/manifest.json [hunk 5]
```diff
@@ -0,0 +1,10 @@
+{
```
Comment (eneo5541) [inline src/components/ux-select/manifest.json:1]: Manifest should be updated from boilerplate to reflect each data property used by the component

File: src/components/ux-select/ux-select.feature [hunk 6]
```diff
@@ -0,0 +1,5 @@
+Feature: ux-select test suite
```
Comment (eneo5541) [inline src/components/ux-select/ux-select.feature:1]: There should be some tests written for this component.

File: src/components/ux-text-box/ux-text.feature [hunk 7]
```diff
@@ -0,0 +1,5 @@
+Feature: ux-text-box test suite
```
Comment (eneo5541) [inline src/components/ux-text-box/ux-text.feature:1]: There should be some tests written for this component.

File: src/components/ux-select/ux-select.hbs [hunk 8]
```diff
@@ -0,0 +1,8 @@
+<label>Select Box
+    <select>
+        <option value="husker">Husker</option>
```
Comment (eneo5541) [inline src/components/ux-select/ux-select.hbs:3]: The selection values are hard coded in? These should come from the datamodel for this component!

**Comment by eneo5541** [comment]
The ux-text-box component should have a manifest.json and use-cases added to it.

**Pull request closed without merging**